keyboards/handwired/sick_pad/config.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
```

Comment:
```suggestion
#define PRODUCT_ID      0xDA20
```

Reply:
Is there any particular for using `0xDA20`?